code/__DEFINES/mobs.dm

```diff
@@ -500,8 +500,9 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define CRUSHER_CHARGE_TANK_MULTI		100
 
 //carrier defines
-#define CARRIER_HUGGER_THROW_SPEED 2
-#define CARRIER_HUGGER_THROW_DISTANCE 5
+#define CARRIER_HUGGER_THROW_SPEED			2
+#define CARRIER_HUGGER_THROW_DISTANCE		5
+#define CARRIER_SLASH_HUGGER_DAMAGE			30
```

Comment:
what the fuck is this, this does more damage than all the xenos slash, except ravager, LoL

Reply:
this is still very high